core/src/main/scala/kafka/log/UnifiedLog.scala

```diff
@@ -248,16 +250,16 @@ case object SnapshotGenerated extends LogStartOffsetIncrementReason {
  *                                  will be deleted to avoid ID conflicts upon re-upgrade.
  */
 @threadsafe
-class Log(@volatile var logStartOffset: Long,
-          private val localLog: LocalLog,
-          brokerTopicStats: BrokerTopicStats,
-          val producerIdExpirationCheckIntervalMs: Int,
-          @volatile var leaderEpochCache: Option[LeaderEpochFileCache],
-          val producerStateManager: ProducerStateManager,
-          @volatile private var _topicId: Option[Uuid],
-          val keepPartitionMetadataFile: Boolean) extends Logging with KafkaMetricsGroup {
+class UnifiedLog(@volatile var logStartOffset: Long,
+                 private val localLog: LocalLog,
+                 brokerTopicStats: BrokerTopicStats,
+                 val producerIdExpirationCheckIntervalMs: Int,
+                 @volatile var leaderEpochCache: Option[LeaderEpochFileCache],
+                 val producerStateManager: ProducerStateManager,
+                 @volatile private var _topicId: Option[Uuid],
+                 val keepPartitionMetadataFile: Boolean) extends Logging with KafkaMetricsGroup {
 
-  import kafka.log.Log._
+  import kafka.log.UnifiedLog._
```

Comment:
Should we rename the logging prefix to UnifiedLog too?

Reply:
Done.